cobc/parser.y

```diff
@@ -6176,7 +6161,7 @@ _data_division_header:
 | data_division_header
 ;
 
-data: DATA { check_area_a ($1); };
+data: DATA { check_area_a_of ("DATA"); };
```

Comment:
Please use DATA DIVISION here and similar `FILE SECTION`, `COMMUNICATION SECTION`, ...